datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -7106,6 +7131,31 @@ input DescriptionUpdateInput {
     subResource: String
 }
 
+"""
+Updates the label of a resource. Currently supports DatasetField labels only
+"""
+input LabelUpdateInput {
+    """
+    The new label
+    """
+    label: String!
+
+    """
+    The primary key of the resource to attach the label to, eg dataset urn
+    """
+    resourceUrn: String!
+
+    """
+    An optional sub resource type
+    """
+    subResourceType: SubResourceType
```

Comment:
These should have ! to be required, since this only applies to DatasetFieldLabels

Reply:
> Overall it's looking in the right direction! I have a couple actionable questions + pieces of feedback. Let me know what you think!

@jjoyce0510 - Thanks for reviewing the PR and providing your feedback. I have couple of items to your feedback. Please check and let me know your thoughts on those and I will work on updating the PR with the updated code asap. Thanks!

Reply:
Addressed.